plugin/pkg/admission/storage/persistentvolume/label/admission.go

```diff
@@ -197,6 +166,56 @@ func (l *persistentVolumeLabel) Admit(ctx context.Context, a admission.Attribute
 	return nil
 }
 
+func (l *persistentVolumeLabel) findVolumeLabels(volume *api.PersistentVolume) (map[string]string, error) {
+	existingLabels := volume.Labels
+
+	// All cloud providers set only these two labels.
+	domain, domainOK := existingLabels[v1.LabelZoneFailureDomain]
+	region, regionOK := existingLabels[v1.LabelZoneRegion]
+	if domainOK && regionOK {
```

Comment:
This is a somewhat significant behavior change. Currently we overwrote the PV labels, and that was ok because it's kubernetes.io namespaced label. Now we are respecting what users set previously which leaves room for errors (e.g. setting the wrong zone) but maybe that's worth the trade-off?